build.xml

```diff
@@ -453,7 +453,7 @@ To get started using Eclipse, execute "ant build-dev" and import the top-level
     </target>
 
     <target name="release-docs" depends="release-javadoc,release-sphinx-api,release-slice2html" description="Generate Docs for all components under dist/docs/api">
-        <zip destfile="${omero.home}/target/OMERO.docs-${omero.version}.zip">
+        <zip destfile="${omero.home}/target/OMERO.javadoc-${omero.version}.zip">
             <zipfileset dir="${dist.dir}/docs" prefix="OMERO.docs-${omero.version}"/>
```

Comment:
See https://ant.apache.org/manual/Types/zipfileset.html - this variable setss the name of the prefix in the archive, in other terms the name of the folder containing the content that gets created when unarchived. We certainly want to have both the archive name and the prefix matching.